_posts/2021-01-29-design-system-crazy.md

```diff
@@ -0,0 +1,137 @@
+When there is all that but nothing is attempted to fix it.
+
+And I strongly agree with this.
+
+I formulated a strong opinion on the situation thanks to the powerful schema his
+article ends with.
+
+![A diagram showing on one side an opposition between code and design, and on
+the other a cycle between features and a Design
+System](/assets/img/design-system-schizophrenia-diagram.png)
+
+It shows that setting-up a Design System for your organization means switching
+from the traditional "designers vs developers" mindset (left) to a Product
+mindset (right).
+
+The target is to have a *team* that owns a *product* (the DS and its artifacts).
+A product that has *customers* (your others teams).
+
+I have to admit that I am not seeing a lot of Design System *teams* in companies
+so far. I see Design System *projects*, for sure. It's very trendy these days.
+They are mainly driven by the base of the organization, ie. some designers
+and/or developers. They work on it on weekends. They come early in the morning
+or they leave late at night to work for the project without slowing down
+production for their "real" job, shipping features.
+
+Sometimes the organization's leadership enters the game, when the project begins
+to see some traction (as it will because, turns out, it's a powerful concept)
+and when the initial volunteers haven't burned out yet. They declare: "We have
+a Design System! It's so good! Thanks so and so". They write a blog post showing
+the world how *their* Design System revolutionized their company. Meanwhile,
+nothing has actually changed inside.
+
+Nothing has changed because the leadership has not embraced that a Design System
+is a product, and nurturing a product requires a dedicated team.
+
+Is such a thing a pipe dream? It should not be, we are already used to this
+paradigm in our engineering departments. We have infrastructure or platform
+teams that cater to the other units, providing a solid platform to run their
+services. We have engineering productivity teams that help others with day to
+day tools (CI, testing environment, etc.). We have analysts and UX researchers
+to support the decision-making of other teams with bespoke data. They are all
+"in-house" services teams.
+
+Organizations willingly invest in all of those but can't form one for the Design
+System? What's the problem?
+
+Is it the idea of mixing designers and developers together as one unit? How is
+it different from what we do for feature teams, squads, swarms or whatever name
+you give to multidisciplinary teams, those that work on features?
+
+Is it an org chart issue? Were it's unclear whether such a team reports to the
+VP of Product or to the VP of Engineering? Why would that matter? Aren't they
+both after the same ultimate company goal?
+
+Is it because designers or developers are already understaffed? Well shoot,
+that's precisely why dedicating a team for it: a Design System is there to help
+you do more with less.
```

Comment:
Je voulais éviter la répétition de "help"

Reply:
```suggestion
Is it because designers or developers are already understaffed? Well shoot,
that's precisely when a dedicated team would help: with a Design System 
they will do more with less, so ensure it becomes more powerful as fast as 
possible.
```

Reply:
Je vais pas mettre le "so ensure it becomes more powerful as fast as possible." qui apporte une injonction qu'il n'y a pas trop dans les paragraphes de cette partie. Et ça permet surveiller le nombre de mots total

Reply:
Yes top !